### DISCUSSION TITLE
Productivity app

### DISCUSSION THREAD

**Discussion by ZikPin** [TomSchimansky/CustomTkinter#1834] [Showcase]
It was my first ever big project (probably not as big for some experienced devs, but hopefully still impressive :> ). I had to learn to use Pickle, git, and pyinstaller and even had to write a custom scrollbar, because the scrollable frame had some issues, and spent over a month working on this project, but finally, it is ready. There are features like option to create to-do lists, habit trackers with different colors, and Pomodoro and stopwatch timers to track how much time was spent working, with stats showing time during the day, week, and month. Screenshots below. If anyone wants to try it out, it is a git repository on my account (hopefully there were no bugs and mistakes while pushing the rep -_-')

![image](https://github.com/TomSchimansky/CustomTkinter/assets/65452275/5ca25e8f-4aca-474b-8f99-5565127ddeb8)

![image](https://github.com/TomSchimansky/CustomTkinter/assets/65452275/a14931b9-00b8-47a1-a3b6-ad1de6dea023)

![image](https://github.com/TomSchimansky/CustomTkinter/assets/65452275/e6a89967-3fb6-457b-a789-7deb15e709f4)

![image](https://github.com/TomSchimansky/CustomTkinter/assets/65452275/76f15f64-fe1b-487b-ab63-517fb1789f88)

**Comment by Akascape** [comment]
@ZikPin Looks awesome. Well made.
It will be great if you release some of the custom-made widgets separately. Like the graph, chart and the calender widget, these will really help others. 
I will also put them in the [tkinter-toolkit](https://github.com/Akascape/tkinter-toolkit)

**Reply by ZikPin** [reply]
Good idea, I'll see what I can do😅

**Reply by avalon60** [reply]
Yes - some pretty cool looking widgets!
